### DISCUSSION TITLE
How to display filtered values on card/tile?

### DISCUSSION THREAD

**Discussion by dpedwards** [Budibase/budibase#4921] [Help]
We are able to filter the right data by options picker and show it in a table as in the first picture shown below. But we need to display the column values of the filtered table on a card aka tile element .
The structure of the elements is shown in the second picture.

Any suggestions on the issue? 
Could it be possible to get the functionality of table directly in a card/tile element?

#1
![MicrosoftTeams-image](https://user-images.githubusercontent.com/2211088/158609605-c64501fa-950a-49f1-b9cd-ef3ab00aa230.png)

#2
![MicrosoftTeams-image](https://user-images.githubusercontent.com/2211088/158573934-b07ad41f-4756-43d7-a203-2fee9dba2b9b.png)

**Comment by mjashanks** [comment]
To dsplay a Card for every row in your `Data Provider`, you need to use a `Repeater`. 

A simplified example would be:

```
> Data provider
   > Repeater
       > Card - bind text to Repeater.TableName.ColumnName
```

**Reply by dpedwards** [reply]
kk thanks for the quick response @mjashanks !

In the meantime we have also found a solution:
We have directly accessed an element from the Data Provider.Row as shown in screenshot 3 and displayed it in the respective card/tile (screenshot 2). We have used the structure below.

```
   Data Provider
   └─── Form
           └─── Container
                        └─── Card
```

#1
![solution1](https://user-images.githubusercontent.com/2211088/158603957-2963d1c2-fae5-4dc7-b21a-dc573460c6b7.png)

#2
![solution2](https://user-images.githubusercontent.com/2211088/158603980-32675bd5-fd2e-4d96-9091-3c6e4171a0b7.png)

#3
![solution3](https://user-images.githubusercontent.com/2211088/158604123-4d344a68-fbe3-48c4-8495-40043851b8ff.png)

**Reply by mjashanks** [reply]
Yep - that works too, though will only work if you want to display one row. The Repeater will.... repeat!

Thanks for sending back this solution though - it's also very valid.
